components/status/StatusAttachment.vue

```diff
@@ -281,6 +283,14 @@ watch(shouldLoadAttachment, () => {
           </div>
         </template>
       </VDropdown>
+      <div v-if="isGif && !getPreferences(userSettings, 'hideGifIndicatorOnPosts')">
+        <button
+          aria-hidden font-bold text-sm
+          rounded-1 bg-black:65 text-white px1.2 py0.2 pointer-events-none
+        >
+          {{ $t('status.gif') }}
```

Comment:
Maybe we don't have to allow "GIF" to be translated? Because this is the proper noun of file format so I think all languages will use "GIF" as is.

Reply:
yes, I think we don't need to translate GIF